README.md

```diff
@@ -243,10 +243,57 @@ globalThis.python.exit = pm.eval("""'use strict';
 """)(sys.exit);
 ```
 
-# Troubleshooting Tips
+# pmjs
+A basic JavaScript shell, `pmjs`, ships with PythonMonkey. This shell can act as a REPL or run
+JavaScript programs; it is conceptually similar to the `node` shell which ships with Node.js.
+
+## Modules
+Pmjs starts PythonMonkey's CommonJS subsystem, which allow it to use CommonJS modules, with semantics
+that are similar to Node.js - e.g. searching module.paths, understanding package.json, index.js, and
+so on. See the [ctx-module](https://www.npmjs.com/package/ctx-module) for a full list of supported
+features.
+
+In addition to CommonJS modules written in JavaScript, PythonMonkey supports CommonJS modules written
+in Python. Simply decorate a Dict named `exports` inside a file with a `.py` extension, and it can be
+loaded by `require()` -- in either JavaScript or Python.
```

Comment:
The description is not very obvious to me. Try add some examples.

```py
# abc.py
import sys # You can use Python libraries.
exports['hello'] = 'world from ' + sys.platform
```

Reply:
Good idea. I create a date lib and showed both JS and Py version